test/extensions/filters/http/fault/fault_filter_integration_test.cc

```diff
@@ -12,6 +12,9 @@ class FaultIntegrationTest : public Event::TestUsingSimulatedTime,
                              public HttpProtocolIntegrationTest {
 public:
   void initializeFilter(const std::string& filter_config) {
+    // There's some deadlock issue with simtime where the alarm may be scheduled
+    // and will not fire. Waiting for the injected delay appears to help.
```

Comment:
This is fixed by #6956 - I'll re-enable it when either this PR or that one lands.